bin/format_exercises

```diff
@@ -1,17 +1,17 @@
 #!/bin/sh
 # Format existing exercises using rustfmt
+set -e
```

Comment:
👍 If we ever do script lints, I'd want to see this line pretty much ubiquitously in our scripts. Saves a lot of errors.